core/common/src/main/java/alluxio/AlluxioURI.java

```diff
@@ -499,4 +502,30 @@ public String toString() {
     mUriString = sb.toString();
     return mUriString;
   }
+
+  /**
+   * Computes the hash of this URI, with SHA-256, or MD5, or simple hashCode().
+   *
+   * @return HEX encoded hash string
+   */
+  public String hash() {
```

Comment:
It would be better to add a unit test method for this `hash()` in `AlluxioURITest`.

Reply:
OK